ruby-challenges.rb

```diff
@@ -10,6 +10,13 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def word_finder(array,letter)
+    array.select{|word|word.include?(letter)}
+    end
+p word_finder(beverages_array,letter_o)
+p word_finder(beverages_array,letter_t)
```

Comment:
Nice!  Interesting approach to omit a do block using curlies.  Make sure to watch the indentation on your end.